packages/react-hooks/src/useAlephBFTCommittee.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017-2023 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ApiPromise } from '@polkadot/api';
+import type { AccountId32 } from '@polkadot/types/interfaces/runtime';
+import type { u32, Vec } from '@polkadot/types-codec';
+
+import { useEffect, useState } from 'react';
+
+import getCommitteeManagement from '@polkadot/react-api/getCommitteeManagement';
+
+import { useApi } from './useApi.js';
+
+export const useAlephBFTCommittee = (session: number): string[] | undefined => {
+  const { api } = useApi();
+
+  const [committee, setCommittee] = useState<string[]>();
```

Comment:
Technically, the type here is:
```suggestion
  const [committee, setCommittee] = useState<string[] | undefined>();
```
I'm surprised that TS has not thrown 🤔 Usually it does when the initial value doesn't match the defined type.

Reply:
it a `useState` magic: 
```
    // convenience overload when first argument is omitted
    /**
     * Returns a stateful value, and a function to update it.
     *
     * @version 16.8.0
     * @see https://reactjs.org/docs/hooks-reference.html#usestate
     */
    function useState<S = undefined>(): [S | undefined, Dispatch<SetStateAction<S | undefined>>];
```
@types/react/index.d.ts

Reply:
Oh wow, I'm pretty sure this wasn't like that some time ago, but I'm glad it is like that now 👍